app/views/admin/works/oh_bio_form.html.erb

```diff
@@ -0,0 +1,149 @@
+          </div>
+          <div class="col">
+            <%= death_form.input :country, priority: ["US","CA"], input_html: {:value => @work.oral_history_content!.interviewee_death&.country },  :as => :country, :include_blank => "(select if none)" %>
+           </div>
+      <% end %>
+    </div>
+
+    <hr/>
+    <h3>Education</h3>
+    <div class="form-group">
+      <%= f.repeatable_attr_input(:interviewee_school, simple_form_input_args: { :label => "" } ) do |sub_form| %>
+        <% if sub_form.object.errors.present? %>
+          <div class="alert alert-danger mr-2">
+            <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+            <%sub_form.object.errors.errors.each do |err| %>
+              <%= err.type %><br/>
+            <% end %>
+          </div>
+        <% end %>
+        <div class="row">
+          <div class="col">
+            <%= sub_form.input :degree %>
+          </div>
+          <div class="col">
+            <!-- TODO make this a proper html5 date input -->
+            <%= sub_form.input :date %>
+          </div>
+          <div class="col">
+            <%= sub_form.input( :institution,  input_html: { data: { "scihist-qa-autocomplete" => qa_search_vocab_path("assign_fast", "all")} }  ) %>
+          </div>
+          <div class="col">
+           <%= sub_form.input :discipline %>
+          </div>
+        </div>
+      <% end %>
+    </div>
+
+    <hr/>
+    <h3>Career</h3>
+    <div class="form-group">
+      <%= f.repeatable_attr_input(:interviewee_job, simple_form_input_args: { :label => "" }) do |sub_form| %>
+        <% if sub_form.object.errors.present? %>
+          <div class="alert alert-danger mr-2">
+            <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+            <%sub_form.object.errors.errors.each do |err| %>
+              <%= err.type %><br/>
+            <% end %>
+          </div>
+        <% end %>
+        <div class="row">
+          <div class="col">
+            <%= sub_form.input :start %>
+          </div>
+          <div class="col">
+            <%= sub_form.input :end %>
+          </div>
+          <div class="col">
+            <%= sub_form.input( :institution,  input_html: { data: { "scihist-qa-autocomplete" => qa_search_vocab_path("assign_fast", "all")} }  ) %>
+          </div>
+          <div class="col">
+            <%= sub_form.input :role %>
+          </div>
+        </div>
+      <% end %>
+    </div>
+    <hr/>
+    <h3>Honors and awards</h3>
+    <div class="form-group">
+      <%= f.repeatable_attr_input(:interviewee_honor, simple_form_input_args: { :label => "" }) do |sub_form| %>
+        <% if sub_form.object.errors.present? %>
+          <div class="alert alert-danger mr-2">
+            <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+            <%sub_form.object.errors.errors.each do |err| %>
+              <%= err.type %><br/>
+            <% end %>
+          </div>
+        <% end %>
```

Comment:
I believe simple_form/kithe should be automatically displaying errors and you shoudln't need to do this manual code. But I'm not totally sure, I could be wrong. Let's compare to the straight work metadata and do it the same.

Reply:
This is correct. Removed